docs/src/configuring-zed.md

```diff
@@ -462,7 +462,7 @@ List of `string` values
 
 - Description: Whether to show the file icon for a tab.
 - Setting: `file_icons`
-- Default: `true`
+- Default: `false`
```

Comment:
It's `false` by default.
https://github.com/zed-industries/zed/blob/2c8ead4423af87a6e0f02788fe40ba176e7588b3/assets/settings/default.json#L439